features/puzzle/puzzle-screen.html

```diff
@@ -61,24 +97,27 @@
 
 
         <div class="center">
-            <p>Countdown for 60 seconds: <span id="countdown"></span></p><!--Experimental timer 60s can be adjusted-->
-            <button onclick="countdown()">Start Timer</button>
+            <p>Countdown for 60 seconds: <span id="countdown"></span></p><!--New countdown timer-->
             <div class="input">
                 <input type="text" id="word" name="word" placeholder="Guess the Answer"><br><br>
-
-                <button class="button1">Submit</button>
-                <button class="button2">Pass</button>
+                <button class="button1" onclick="setUserInput()">Submit</button> <!--Buttons for game operations-->
+                <button class="button2" onclick="testCall()">Pass</button>
+                <button class="button3">Reset</button>
+                <button class="button4">Give Up</button>
             </div>
         </div>
     </div>
 
 
 </body>
 <script>
+
```

Comment:
move all javascript code within the script tag into its own file called puzzle-screen-script.js

Reply:
made a dedicated JS file just now and moved it there